easybuild/easyconfigs/k/Kent_tools/Kent_tools-337-linux.x86_64.eb

```diff
@@ -0,0 +1,32 @@
+#This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#Author: Ravi Tripathi
+#Email: ravi89@uab.edu
+
+easyblock = "PackedBinary"
+
+name = 'Kent_tools'
+version = '337'
+versionsuffix = '-linux.x86_64'
+
+homepage = 'http://genome.cse.ucsc.edu/'
+description = """Kent tools: collection of tools used by the UCSC genome browser."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download from http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/, and pack into tarball with:
+#   mkdir Kent_tools
+#   rsync -aP rsync://hgdownload.cse.ucsc.edu/genome/admin/exe/linux.x86_64/ Kent_tools
+#   tar cfvz Kent_tools-20130806.tar.gz Kent_tools
+source_urls = ['http://hgdownload.soe.ucsc.edu/admin/exe/']
+sources = ['userApps.v%(version)s.src.tgz']
```

Comment:
@boegel Well.....it is kinda building from source here, so it should be for all platforms. I will remove it.

Also I gave the name for this Easyconfig as kent_tools since there was already an Easyconfig available by that name and I wanted to keep this consistent, but most of the folks know it by ucsc tools, so it might be difficult for them to search for it if they just search for ucsc. Your suggestions?